wallet-new/README.md

```diff
@@ -188,14 +188,11 @@ $ nix-build -A walletIntegrationTests --arg useStackBinaries true
 > test, you can pass the `--check` flag to force the test to run again. `--check`
 > is used to confirm that results from one test match the results again.
 
-Wallet integration tests can be used also with seed/match options that behave like hspec test runner (and are passed to).
---match PATTERN - behave exactly like Hspec's --match allowing to run only a subset of tests
---seed SEED - enable passing an external, predictable seed to the test runner
-Example allowing the use of concrete seed and testing Address related tests only:
-
+The wallet integration tests support --hspec-options to pass options to HSpec. 
+For example:
 ```
 $ nix-build -A walletIntegrationTests -o launch_integration_tests
-$ ./launch_integration_tests --match 'Address' --seed 47286650
+$ ./launch_integration_tests --hspec-options '--match Address --seed 47286650'
```

Comment:
I'd like to understand the rational behind this change. Do you actually need to pass around hspec options that aren't `--match` or `--seed` ? If so, please add this rational as part of the commit's message.